build.xml

```diff
@@ -182,52 +182,44 @@
     <!-- Create demo users. -->
     <target name="create-demo-users" description="Create demo users.">
         <foreach list="${drupal.demo.users}" param="drupal.demo.user" target="create-demo-user" delimiter="," />
-        <!-- Create a user with only "authenticated user" role assigned. -->
+        <foreach list="${drupal.demo.authenticated_users}" param="drupal.demo.user" target="create-demo-authenticated_user" delimiter="," />
+    </target>
+
+    <!-- Create a demo user. Subtarget of "create-demo-users". -->
+    <target name="create-demo-user">
```

Comment:
Why all these changes? Nothing is mentioned in the ticket about changing the demo users that are created by Phing. It also doesn't seem like anything really changes, there is still one user created, but now there are two targets that do almost the same thing.